### PR TITLE
flesh out seed command bones

### PR DESCRIPTION
**Pull request by chrichri17**
This starts resolving #22

## Usage
Run in your terminal `python manage.py seed --mode=clear`.

Or `python manage.py seed --help` to display the usage

## What's next ?
- [x] create a data dir with dummy quizzes and questions.
- [ ] when mode is refresh, read the data and seed the mongodb

### CODE REVIEW

**Comment by chrichri17** [comment]
Current dependencies on/for this PR:
* main
  * **PR #54** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/54" target="_blank">Graphite</a> 
    * **PR #55** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/55" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/YacineSteeve/quizoot/55?utm_source=stack-comment).

File: backend/api/management/commands/seed.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand
+from argparse import ArgumentParser
+
+from api.client import client
```
Comment (YacineSteeve) [inline backend/api/management/commands/seed.py]: Does it work with this import ?
I think it should either be `from ....api.client` (not really like this personally) or `from backend.api.client`

File: backend/api/management/commands/seed.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+MODE_CLEAR = "clear"
+MODE_REFRESH = "refresh"
+
+
+class Command(BaseCommand):
+    help = "seed database for testing and development."
+
+    def add_arguments(self, parser: ArgumentParser):
+        parser.add_argument("--mode", help="Specify the mode (clear | refresh)")
```
Comment (YacineSteeve) [inline backend/api/management/commands/seed.py]: A little suggestion:
```python
parser.add_argument("--mode", help=f"Specify the mode ({MODE_CLEAR} | {MODE_REFRESH})")
```

**Review by YacineSteeve — APPROVED** [review]
It could be helpful to add a target to `Makefile` when the command is ready.
Looks good 💯

File: backend/api/management/commands/seed.py (hunk 1 above)
Comment (chrichri17) [reply backend/api/management/commands/seed.py]: Yeah it worked. `backend.api.client` is good too.

File: backend/api/management/commands/seed.py (hunk 2 above)
Comment (chrichri17) [reply backend/api/management/commands/seed.py]: Nice. Will do

File: backend/api/management/commands/seed.py (hunk 1 above)
Comment (chrichri17) [reply backend/api/management/commands/seed.py]: Edit:

I checked again and now it's not working anymore. That's weird...

Thks for spotting this

**Pull request merged**